### DISCUSSION TITLE
configuration documentation

### DISCUSSION THREAD

**Discussion by Rhodes-Scholar** [robotshell/magicRecon#4] [General]
Do you have any documentation on the values that need to be in the configuration file?

**Comment by robotshell** [comment]
Hi Sara,

There is no documentation on the configuration file values because it is very simple. 

- In value *dnsDictionary* you have to put a dictionary to perform a subdomain search, for example, you can put the following dictionary: 
https://github.com/danielmiessler/SecLists/blob/master/Discovery/DNS/dns-Jhaddix.txt
- In value *dictionary* you have to put a dictionary to perform a directories and files search, for example, you can put the following dictionary: 
https://github.com/danielmiessler/SecLists/blob/master/Discovery/Web-Content/dirsearch.txt
- In value *github_token* you have to put your GitHub Personal Access Token, you can create it by following the steps below: 
https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token
- In value *burpCollaborator* you have to put your Burp Collaborator link or any other alternative like https://requestbin.net.

Remember that you have to enter the path where you have the files, for example:
```
dictionary=/home/kali/tools/SecLists/Discovery/Web-Content/dirsearch.txt
```

If you have any other questions, don't forget to ask. 

Best regards!

**Reply by Rhodes-Scholar** [reply]
Thanks!
